src/main/resources/hudson/plugins/sauce_ondemand/PluginImpl/config.jelly

```diff
@@ -12,9 +13,70 @@
             <f:entry title="${%Override Sauce Connect Path}" field="sauceConnectDirectory">
                 <f:textbox id="directoryBox"/>
             </f:entry>
-            <f:entry title="${%Sauce Connect Options}" field="sauceConnectOptions">
+            <hr/>
+            <div id="validationMessage" style="color: red;"></div>
+            <div id="deprecationMessage" style="color: orange;"></div>
```

Comment:
Is it possible to have the deprecation message right above the Sauce Connect Legacy Options text box?

Reply:
Yes, fixed in be523b2